src/HotChocolate/MongoDb/src/Data/Execution/IMongoExecutable.cs

```diff
@@ -0,0 +1,13 @@
+using HotChocolate.Data.MongoDb;
+
+namespace HotChocolate.Data.MongoDb.Execution
+{
+    public interface IMongoExecutable : IExecutable
```

Comment:
public interfaces need documentation. We should do this now when we touch code or create new code.